science/mpich/Portfile

```diff
@@ -113,9 +93,55 @@ subport ${name}-default {}
 subport ${name}-devel {}
 subport ${name}-devel-default {}
 
-# Linker for Apple clang version 421.11.66 segfaults
-# See https://trac.macports.org/ticket/36654#comment:9
-compiler.blacklist-append   {clang >= 421.11.66 < 425.0.24}
+if {${name} ne ${subport}} {
```

Comment:
This is one place that should exclude the obsolete subports.

Reply:
Done